src/platform/Linux/ThreadStackManagerImpl.cpp

```diff
@@ -204,11 +222,15 @@ CHIP_ERROR ThreadStackManagerImpl::_SetThreadProvision(ByteSpan netInfo)
 
 CHIP_ERROR ThreadStackManagerImpl::_GetThreadProvision(ByteSpan & netInfo)
```

Comment:
I think we can add GetThreadProvisionin test in existing cirque test ? @erjiaqing

Reply:
We may never use this API in CHIP Thread provisioning, if not used, please consider to remove.